doc/introduction/photonic_hardware.rst

```diff
@@ -0,0 +1,95 @@
+Photonic hardware
+=================
+
+Using Strawberry Fields, you can submit quantum programs
+to be executed on photonic hardware using the Xanadu cloud platform.
+
+
+Configuring your account
+------------------------
+
+Before using the Xanadu cloud platform, you will need to configure your credentials. This
+can be done using several methods:
+
+* **Configuration file (recommended)**. Use the Strawberry Fields :func:`~.store_account` function,
+
+  >>> sf.store_account("AUTHENTICATION_TOKEN")
+
+  or the ``sf configure`` :doc:`command line interface </code/sf_cli>`,
+
+  .. code-block:: bash
+
+      $ sf configure --token AUTHENTICATION_TOKEN
+
+  to store a configuration file containing your credentials. Replace
+  ``AUTHENTICATION_TOKEN`` above with your Xanadu cloud access token.
+
+* **Environment variables**. Set the ``SF_API_AUTHENTICATION_TOKEN`` environment variable
+  with your API token.
+
+* **Keyword argument**. Pass the keyword argument ``token`` when initializing the
+  :class:`~.RemoteEngine`.
+
+.. seealso::
+
+    For more details on configuring Strawberry Fields for cloud access, including
+    creating local per-project configuration files, see the :doc:`Advanced configuration options </code/sf_configuration>`, and :func:`~.store_account` function.
```

Comment:
```suggestion
    creating local per-project configuration files, see the :doc:`configuration </code/sf_configuration>`, module and :func:`~.store_account` function.
```
Just because "Advanced configuration options" seems to be a page that doesn't exist